README.md

```diff
@@ -232,4 +232,8 @@ _Immutable Web Applications_ methodology builds on several trends in web applica
 - Examples
 - Advocacy
 
+## Acknowledgements
+
+_Immutable Web Apps_ was developed based on the research and experiences building web applications at [Meltwater](https://www.meltwater.com/). Learn more about the things we are doing at our engineering blog: [_under the hood_](https://underthehood.meltwater.com/).
```

Comment:
```suggestion
_Immutable Web Apps_ was developed based on the research and experiences building web applications at [Meltwater](https://www.meltwater.com/). Learn more about the things we are doing at our engineering blog: [underthehood.meltwater.com](https://underthehood.meltwater.com/).
```

Reply:
I would like to get the full blog URL into the post. No special reason, I just find that it looks better :)